components/home/partners/logos/Chipax.tsx

```diff
@@ -0,0 +1,31 @@
ETHJBNq16SGCgOyg7bdE+CXd20aSmAYyx8Ri/Ld7bL8IZHqOQt9koIguJaF2ppkWrASOOifi2SESINBzg7xQ1wvAIA72XK7xhhdEGaI8U1NAXgLqkhbh65Y84h4RaPRuDCKomdaa58+aCOiy6Io+jZEvIbnLaRdgyOi7zVmeEHsya5r+JnzVwS7yb9W3nd/QSbOkVtUw25qDZHGrHoCIvxyVgIjWf7TFP6+EtzusE8aXpn/8sGDB8UK3PA8gTutta+VQqM0isXi5Yi4JEVuAODK5bIjojf6x9gCVQA5KwAv/NPLjOIkgI+Nu0+ZR7Cnei0NXU0u78qNN974RER8gIsjZS7q/9LWOfa8hQ973kKHn5crhz32jsO3gH0l4FvA26EE4FSKaC6ewCxZ/rM44bsVieyLD7ASaHOst10qlRy/aPdaa48AwDs4Q+RNRPSniPhxROxKcR1bZccOHTrkEPGjfIxtXRReFcCpdzIIgmeA19qZJ1KvNGazVYo0nRu0FQoFv922rEmwxvMCjw3bpyet/bNY/4j4W/53vPMQBXCZnAcRtfNarWzu8HK8z/AEtksJ5OgJ9LX8MSXbZzvCPrNq+fskFHYljmPHueFtDgutF4tFF8exazQaLo5jV6/XXaPRcCxAHACsAS/UvbKy4hDxDklbHPK69/UAxlmH1nsPDokCYME9kQIgot/zFQCc3rNpZhWAFz5Bv/CLiP59nP3J2Fqt1j5EvEdqCfja/JF/zDTk/BcXF3+cw0hdTyF9So6Rdg3SMobw1NrAmbemmCvSisV2oiegln+2JO7VqxHx/kajIX2JuhwaOo6IxwHgBGy2JJC/rwdB4PiadxGxkLbfLY7fe9kB4G9ZGD7KwvEGY8b3AFhQP8L56jX/82HxBOgN/Kw9ymP8uIzfjKEArLVXs9A7xhWumSsAuQdhGF6IiN/gidtjbKS82T+/UfAs+HfxM/8YewH3DWoPkWhH8q/SjwpONaJ7gXze53z83kSyROQJnovoRlGUeZvqeaNvsdiseAIwWYroXKZ6zsLaCIPwlSoRnUtEv4uIXygWi65er7s4jl2tVnPVatVVq1W3tLTUaz+OiA8CQOz3kB81C0iuJSLeceTIEReGobvlllscAFxnzGhCyhNOzaNHj7ogCNzRo0cd8hKCEyiAt91yyy0uDEN35MgRh7yIz6gFWd51vurQoUMuiiLHYbPMFYDnDV1z8803u2KxKF7aV+M4fvy4x/IzyYjo/xqNhouiyB0+fNgR0VX+sdPGQ0RXHTlyxJVKJVcul93hw4cdAFSM2fpee17AT0sr9yiK3G233eYQ8dAw+9jp9G0bMc8Tw2mWf7+wT97FITvB8k8j+dKGYfj8IAiuB4DDiPjXiPgPiPh3uNmE7UYiukragBvTO9ex16UlosuI6CpEvAIRrwyC4En+56Ps68CBA5cQ0VVBELwsDMOflyK3UZ8Lr33CE621L0fEK4joKlF44+6PiM4FgJdYa1/O+/vecfY3zLGCIHgSX9MrAOAVMvZJ3n35LQA8BwBegYhXBEHwMgn/DfIAuBDxJUEQvFyu6ShdTb2Cthf69zgIgu8adh87ndQU0VnwBMZpIDcPRV7zEPMfhkKhsGfUcU9aoWpM/5d2TAs1s33lsT/TR0nmlP+fus8s3pF++xh0HuN+lsWxdxs7pVhs5ou8OFY+t5Z/PyTv3zm3V66tVKpKXUDWGSt+vcEk+xZFJvua9PmWuhtvbJO+Lwurq6tnyf7yjFvnMPYe/rMw7PMgBp2/jXOv5XzyOK8dwSwXi7HgHjgxPEsTvvNS5JUjC3mH1hRFyZ6Z9QQGzQkUCoV9M17ktWPCPoqi7GD6FYtthyeQTBFN8wQkRY2/M5OW/7xP+CqKssuY1WKx5JwAC/zXW2t/m8c1E5b/TpvwVRRlFzHLxWIJT8CFYejCMJQF3FMXc9EiL0VRlNGYyRRRRDx08ODB3iLpcKoX+QYAdFdWVrbb8k9t7DbrRV6KoiinkeYJzEKxGCKWuTy8Y611iLhhre1yqbwsWbjrVvJSFEXJmpnxBGTRBwC4rlwu9xQAAGxYa7u8CMSvGLPZuTCvcYzS3kGFv6Io8852p4gueBPTN3Af+g3eOpJjb6113Nhr7AZWW7HbirwURVGM2SZPoF9vHxH+5XJZOlLKOqW5rSw2yPLHGVjGUVEUJU+m7QkMbO+wvLzsAOAAAFQOHjyY68pig4q8VPgrirIrmFax2JDtHUryfVnYY6u2EWOOZcv2Dhr2URRl1zCoWCwDT2Doxm6tVmuvZPuM0UBuS7S9g6IoSoJ+nsCkcwKjNHbzOk/msrKYpnoqiqIMIOOJ4b7Cf6s1fLNeWUzbOyiKomxBv2IxUQLDhoOSlj+neqYu5tJvP316B43sCajlryiKMjwTpYhmuZLXpJ7ATlrDV1EUZVqkNpDbamJ4lDV8h23vMO7KYmr5K4qijM9IK4u1Wq29ea3hm1xZbKs1hpPCX4u8FEVRRmekYjFj8lvDdxRPQNAiL0VRlAnwwzr9PAFEXDLGGGvtDWkTvlmt5LXVGsO1Wq2nBADg1l24hq+iKEr2DPAEuqVSyVlrP4mIDgBcP8s/i55CgzwBPn4XAP7R6zCqqZ6KoiiTMKhYzFrbZYEri7rkupJXv5XFAKBrre01lNMJX0VRlAwZ0DaiAwAbWxV5ZcWAFNENtfwVRVFyYJg1hvOy/FPGkhoO0sZuiqIo+XFaiihPvnamYfknSSqBOI6dtbarRV6Koij54XsCb202m46IQmPyX8M3SXJOYHl52SHitcao5a8oipIXvRAPIr5Y/tMXyNNCUkRXV1fPstb+pP9/iqIoSk74An+aln8SP+Skwl9RFGVKOOf2GmO2Tfh741jgsSiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKoiiKMvf8P/b5+KnIpVAGAAAAAElFTkSuQmCC"
```

Comment:
What's going on here?  Embedding a PNG into a TSX component?  Similar things above with SVGs

Reply:
Two different things:
1. I copied this from Figma. Can probably get it better elsewhere, but holding until after Andy and Brian chat.
2. Rather than import an svg or img, since svg's are just html, I threw it into a tsx file and it's bundled into the app.